src/platform/plugins/shared/workflows_execution_engine/server/execution_functions/setup_dependencies.ts

```diff
@@ -28,6 +28,10 @@ import { WorkflowExecutionState } from '../workflow_context_manager/workflow_exe
 import { WorkflowEventLogger } from '../workflow_event_logger/workflow_event_logger';
 import { WorkflowTaskManager } from '../workflow_task_manager/workflow_task_manager';
 
+const defaultWorkflowSettings: WorkflowSettings = {
+  timeout: '10m',
```

Comment:
I’d suggest setting the default timeout to 1d

Reply:
@talboren @shahargl 
What do you think? Should we align default workflow timeout with task manager task timeout?
I used 10m value because we spoke about in that thread - https://elastic.slack.com/archives/C08U04SUN49/p1761314394268429